### PR TITLE
[bitnami/mediawiki] Add env var to control skipping config validation during startup

### PR DESCRIPTION
**Pull request by iain-ilearner**
As suggested/requested in https://github.com/bitnami/containers/issues/65721#issuecomment-2078888820 here's a PR introducing a new environment variable that controls whether or not validation is performed of configuration file during database upgrade at container start-up.

This allows the use deprecated values (such as `$wgAllowImageTag`) in LocalSettings.php

This PR also adds a log message after connecting to the database to better indicate which part of the start-up process is running.

### CODE REVIEW

**Review by dgomezleon — COMMENTED** [review]
Thanks a lot for contributing. I suggested some minor changes.

**Comment by iain-ilearner** [comment]
Thank you for taking time to look at this PR, but I'm unable to see any of the minor changes you referred to. Are they in this PR on Github?

**Review by dgomezleon — COMMENTED** [review]
Sorry @iain-ilearner .
It seems the suggestions were not submitted.

File: bitnami/mediawiki/1/debian-12/rootfs/opt/bitnami/scripts/libmediawiki.sh [hunk 1]
```diff
@@ -151,7 +151,12 @@ mediawiki_initialize() {
         db_port="$MEDIAWIKI_DATABASE_PORT_NUMBER"
         mediawiki_wait_for_db_connection "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
         # Perform MediaWiki database schema upgrade
-        debug_execute php "${MEDIAWIKI_BASE_DIR}/maintenance/update.php"
+        info "Performing database schema upgrade if needed"
+        update_args=""
+        if is_boolean_yes "$MEDIAWIKI_SKIP_CONFIG_VALIDATION"; then
+            update_args+=" --skip-config-validation"
+        fi
+        debug_execute php "${MEDIAWIKI_BASE_DIR}/maintenance/update.php" $update_args
```
Comment (dgomezleon) [inline bitnami/mediawiki/1/debian-12/rootfs/opt/bitnami/scripts/libmediawiki.sh:159]:
```suggestion
        info "Performing database schema upgrade if needed"
        local -a update_args=()
        if is_boolean_yes "$MEDIAWIKI_SKIP_CONFIG_VALIDATION"; then
            update_args+=( "--skip-config-validation" )
        fi
        debug_execute php "${MEDIAWIKI_BASE_DIR}/maintenance/update.php" "${update_args[@]}"
```

**Review by dgomezleon — COMMENTED** [review]
Thanks for applying the changes. I also detected that `update.php` is called in line 127 (first intallation), where this validation could be added too.

Could you please take a look?

**Comment by iain-ilearner** [comment]
The reason I didn't add it on line 127 initially was because line 95 checks if we're in an empty dir before doing the install/upgrade. 
https://github.com/bitnami/containers/blob/79ff6eb1aa0e0720630ace0e58eeddc33ed90c5e/bitnami/mediawiki/1/debian-12/rootfs/opt/bitnami/scripts/libmediawiki.sh#L95

I'm not sure how you could get a `LocalSettings.php` that fails validation at this point but I guess with some weird mounts it might be possible, so I've added the skip config env var here too.

**Review by dgomezleon — APPROVED** [review]

**Pull request merged**